components/profile/ProfileDetail.vue

```diff
@@ -732,7 +732,7 @@ useAsyncData('tabs-count', async () => {
 })
 
 const fetchTabsCountByNetwork = async (chain: Prefix) => {
-  const account = id.value.toString()
```

Comment:
shouldn't you keep `.toString()` ?

Reply:
no, if so there would be a Ts error, id is already :
```
const id = computed(() => route.params.id.toString() || '')
```